src/main/java/core/basesyntax/service/impl/CsvFileReader.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.service.impl;
+
+import com.opencsv.bean.CsvToBeanBuilder;
+import core.basesyntax.model.FruitDto;
+import core.basesyntax.service.FileReader;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.List;
+
+public class CsvFileReader implements FileReader {
+
+    @Override
+    public List<FruitDto> readData(String fileName) {
+        try (java.io.FileReader fileReader = new java.io.FileReader(fileName)) {
```

Comment:
```suggestion
        try (FileReader fileReader = new FileReader(fileName)) {
```

Reply:
I should rename interface to avoid this situation